macros/seed_data_helper.sql

```diff
@@ -0,0 +1,13 @@
+{% macro seed_data_helper(seed_name, warehouses) %}
+
+{% if target.type in warehouses %}
+    {% for w in warehouses %}
+        {% if target.type == w %}
+            {{ return(ref(seed_name ~ "_" ~ w ~ "")) }}
```

Comment:
I came across a weird *bug* where I would see the macro fail without it? I am not entirely sure if it is needed, but felt it was safer to include in case the *bug* appeared again.